### PR TITLE
Support for Blender 3.0

### PR DESCRIPTION
**Pull request by nagadomi**
Merge from https://github.com/UuuNyaa/blender_mmd_tools/pull/8

I think it's better to keep this version available for a while, so I'll maintain only the Blender compatibility issues.
I will merge this change with collaborator privileges in a few days if possible.
If there are any problems, please let me know.

### CODE REVIEW

**Comment by UuuNyaa** [comment]
Since the two forks are mixed up and users are confused, I would like to change the dev_test head to something that warns like [warning branch](https://github.com/UuuNyaa/blender_mmd_tools/tree/warning).

In fact, on my Discord server, I am **often** asked about problems caused by using the powroupi version (v0.7.x).

@nagadomi What do you think?

**Comment by nagadomi** [comment]
@UuuNyaa 
You already added the message 
>This repository is NOT maintained!
>The latest version is available in [UuuNyaa/blender_mmd_tools](https://github.com/UuuNyaa/blender_mmd_tools).

They don't read README, so any description is no good.
Also, setting the default branch to something other than master is not recommended, as the repository URL will not be found in Google search results due to github.com's robots.txt.
The reason why sugiany version is still used is because sugiany/blender_mmd_tools is the top Google search result for "mmd_tools".
So all you need is to get the top of the search results for "mmd_tools", I think.

**Comment by UuuNyaa** [comment]
@nagadomi Thanks for your reply.

I will leave only two files in the dev_test branch instead of writing something in the README.
One is the README file and the other is the correct download URL file.

This way, if a user accidentally downloads a zipped dev_test branch, they will not be able to install it.
(and they will see correct download URL and README files)

It's the same reason why the default branch name in my repository is **warning**.
Some users try to install it the wrong way and ask me how to fix it.

However, I may change the default branch because of the robot.txt problem you told me about.

> So all you need is to get the top of the search results for "mmd_tools", I think.

I think it is very difficult and needs time...

**Comment by nagadomi** [comment]
Also, typical Blender users do not update addons even when they upgrade Blender. So they keep copying and using the old addons. I think it's also the reason.
Also, changing the contents of the default repository is not a good idea because it breaks systems that use git clone, such as Docker.

I also think you should allow forked repository. If I fork your repository, do you want me to stop doing it?

**Comment by UuuNyaa** [comment]
@nagadomi Thank you for your suggestion.

You're right. I will continue to encourage users to migrate.
I thought the timing of the upgrade to Blender 3.x was a good time to migrate users, but it was put off.

**Comment by nagadomi** [comment]
@UuuNyaa
The reason I maintain this version is that the community has many of its own MMD models made in powroupi version of mmd_tools.
In the future, any changes that break compatibility, such as property renaming, will break all of these.
However, I think there are some issues that cannot be fixed while maintaining full compatibility.
So I thought it would be safer to have the classic version of mmd_tools.
If you have continuity in your development, it might be a good idea to create a `classic` branch in your repository and move it from here.

Basically, it is better to consider compatibility and patching, but it would be a limitation to the speed of your changes.
